source/agora/consensus/data/Block.d

```diff
@@ -551,18 +555,54 @@ public Block makeNewBlock (Transactions)(const ref Block prev_block,
     return block;
 }
 
+/***************************************************************************
+
+    Create the block random seed from the provided pre-images
+
+    It might be that not all validators provide their `pre-image` and in
+    that case the length of the preimages can be less than the number of
+    active validators.
+
+    Params:
+        preimages = preimages sorted by the utxo keys of the validators
+
+    Returns:
+        Hash of all provided pre-images or Hash.init if an exception is thrown
+
+***************************************************************************/
+
+mixin AddLogger!();
```

Comment:
This should be below the imports. The ddoc string is documenting the mixin instead of `createRandomSeed`.

Reply:
Moved up.